### DISCUSSION TITLE
v4 Question

### DISCUSSION THREAD

**Discussion by DrMxrcy** [TGRHavoc/live_map#98] [Q&A]
Hello! Is v4 Ready for Testing?

**Comment by TGRHavoc** [comment]
Unfortunatley it's not.
After V4 of the resource is done, the interface will need to be written in a way that we can use Server-Sent Events (SSE). This part _should_ be fairly easy but, I'm wanting to re-write the interface to be a bit cleaner (and hopefully work on accuracy while I'm at it).
The main hurdle I was having issues with is done, as in the SSE on the resource side, so it's mainly a matter of getting feature parity with what was available before.
